flickr.py

```diff
@@ -25,18 +25,19 @@ def params_unique_combination(baseurl, params_d, private_keys=["api_key"]):
             res.append("{}-{}".format(k, params_d[k]))
     return baseurl + "_".join(res)
 
-def search_flickr_by_tags(tags):
+def search_flickr(data):
```

Comment:
So you renamed the function to reflect that it's more general purpose? That makes sense.